Extensions/retags.js

```diff
@@ -1,6 +1,6 @@
 //* TITLE       Retags **//
 //* DEVELOPER   alexhong **//
-//* VERSION     0.6.9 **//
+//* VERSION     0.7.0 **//
```

Comment:
actually changed my mind. just push this to 1.0.0. we should have done that before anyway